### ISSUE TITLE
Modsuits and The Loss To Miners 

### ISSUE THREAD

**Issue by mixzombie** [tgstation/tgstation#63624]
Look at the Old Mining Hardsuits, They could be purchased out of a mining vendor making it very convenient. Also the Mining Hardsuit also started off with Melee Defense, AND could be upgraded with Goliath Plates to enhance the hardsuit making it more useful to miners. A fully upgraded Mining Hardsuit offered 7 Melee Defense, As much as Drake Armor or Fully Upgraded Explorers Suit. Now has Zero!!! 

The new "goonsuit" (the term i will forever use to describe these "new" hardsuits) out of the vendor Is cheaper but takes Much Much longer to get.  Unlike Every Other Department whose hardsuits have just changed to the updated "goonsuits" Mining Vendors have been stripped of ther Hardsuits and require going to Robotics for them to Make the "goonsuit". 

**TLDR; Miners Cant Upgarde Armor On "Goonsuits" and Unlike Every Other Department that had hardsuits Miners Can No Longer Directly get ther "Goonsuit" they have to go to robotics for it**

**Comment by Fikou** [comment]
this is not a bug

**Comment by mixzombie** [comment]
Wasn't a Bug report, Feature Request. And a pretty reasonable one at that. Thanks

**Comment by Mothblocks** [comment]
The issue tracker is only for bug reports, please use the forums for feature requests.

**Comment by Mothblocks** [comment]
![image](https://user-images.githubusercontent.com/35135081/147425064-f87fa158-f7c3-4301-b8ae-b4a92a89f9f7.png)

![image](https://user-images.githubusercontent.com/35135081/147425070-cc331769-eff0-4f6e-9362-94071f18232f.png)
